docker/centos7/setup.sh

```diff
@@ -1,623 +0,0 @@
-#!/usr/bin/env bash
```

Comment:
I couldn't find a place where this script is used, so I think we can remove it.

Reply:
This is correct, is is not used in the repo. It is used to setup the release EC2 instance, and _should_ align with the Dockerfile. It should stay.

Reply:
Good to know, let me add a comment to that script. I wonder if using the script inside the Docker image is a better idea to keep those files aligned e.g. currently they already diverge by installing go 1.18 and go 1.17.